library/include/DataIdentity.h

```diff
@@ -393,6 +391,10 @@ namespace df
     };
 #endif
 
+    type_identity* lua_touserdata_(lua_State* state);
```

Comment:
are the trailing underscores necessary? can we not just fully-qualify the names when needed (or would that require changing too many places)?